changelog/changelog.py

```diff
@@ -170,6 +186,33 @@ def default_changelog_generator(items):
     return lines
 
 
+def default_header_changelog_generator(items, labels, line_generator):
+    """
+    Contruct the default changelog for the header labels.
+    """
+
+    lines = []
+    label_sorted_items = {}
+    for item in items:
+        for label in item['labels']:
+            label_name = label['name']
+            if label_name in labels:
+                if label_name not in label_sorted_items.keys():
+                    label_sorted_items[label_name] = [item]
+                else:
+                    label_sorted_items[label_name].append(item)
```

Comment:
There's a nice way of doing this with `dict.setdefault`:
```suggestion
                label_sorted_items.setdefault(label_name, []).append(item)
```